README.md

```diff
@@ -139,6 +139,11 @@ with:
 continue-on-error: true
 ```
 
+**You will need to use `pull_request` as the pipeline trigger to ensure that
```

Comment:
As per code reviews, it's better not to make documentation personal
```suggestion
**`pull_request` must be used as the pipeline trigger to ensure that
```

Reply:
Makes sense. I've updated it now, and after reviewing I added a few more details to further clarify.